src/main/java/codeu/controller/BotServlet.java

```diff
@@ -142,6 +154,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
         messageStore.addMessage(message);
 
+        String responseMessageContent;
+        try{
+            responseMessageContent = botService.process(cleanedMessageContent);
+        }
+        catch(IOException | HttpException ex){
+            responseMessageContent = "Oh, it seems we could not reach EastBot right now! Please try again soon, or let an admin"+
+                    " know if the issue persists!";
+        }
+
+        User chatbot = userStore.getUser("EastBot");
```

Comment:
This username will be taken upon server startup, so we know it will be there, and we dont have to worry about a regular user taking it up

Reply:
Could you please clarify what you mean by this? For example, does it mean that "regular" users won't be able to use the username "EastBot"?

Reply:
Could you please clarify what you mean by this? For example, does it mean that "regular" users won't be able to use the username "EastBot"?

Reply:
Yup, exactly c: